nodes/arduino_handler.py

```diff
@@ -383,6 +383,10 @@ def connect_serial(serial_connection=None):
             trace('arduino_handler serial write %d bytes: >%s<', len(message), message.replace('\n',''))
             # Read
             buf = serial_connection.readline()
+            # handle reading nothing, happens when serial port times out
```

Comment:
OK, I will remove the check from here.  The other thing I was doing is sleeping a bit, but I don't think that matters.